DEVELOPMENT.md

```diff
@@ -65,64 +76,29 @@ EOF
 ### Running Unit Tests
 
 ```bash
-go test ./pkg/...
+make unit
 ```
 
 ### Running End-to-end Tests
-```bash
-go test ./test/...
-```
 
 * To run the e2e tests, kpack must be installed and running on a cluster
-
-* The KPACK_TEST_NAMESPACE_LABELS environment variable allows you to define additional labels for the test namespace, e.g.
-
-```bash
-export KPACK_TEST_NAMESPACE_LABELS="istio-injection=disabled,purpose=test"
-```
-
-* The IMAGE_REGISTRY environment variable must point at a registry with local write access 
+* 🍿 These tests can take anywhere from 20-30 minutes depending on your setup
 
 ```bash
-IMAGE_REGISTRY=gcr.io/<some-project> go test ./test/...
+IMAGE_REGISTRY=gcr.io/<some-project> \
```

Comment:
Side note, while `./hack/apply.sh` works with index.docker.io I couldn't get the end-to-end tests to pass this way (they failed with "unauthorized" error). I recall there are some oddities WRT ggcr/docker and Docker Hub specifically, but I didn't dig too much here.

Reply:
ya there are some weirdisms but I forget the exact config needed for running the e2e tests against dockerhub